packages/protocol/script/DeployOnL1.s.sol

```diff
@@ -36,9 +34,11 @@ import { P256Verifier } from "p256-verifier/src/P256Verifier.sol";
 /// @notice This script deploys the core Taiko protocol smart contract on L1,
 /// initializing the rollup.
 contract DeployOnL1 is DeployCapability {
-    uint256 public constant NUM_GUARDIANS = 5;
+    uint256 public constant NUM_MIN_MAJORITY_GUARDIANS = 7;
+    uint256 public constant NUM_MIN_MINORITY_GUARDIANS = 2;
 
-    address public constant MAINNET_SECURITY_COUNCIL = 0x7C50d60743D3FCe5a39FdbF687AFbAe5acFF49Fd;
+    address public constant MAINNET_SECURITY_COUNCIL = 0x7C50d60743D3FCe5a39FdbF687AFbAe5acFF49Fd; // council.taiko.eth
```

Comment:
Seems `security council` is also not used.

Reply:
Removed: https://github.com/taikoxyz/taiko-mono/pull/16913/commits/369038df4158daa70f2a0488ab78bf32215e69c2